compose/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "greenpress-docker-scripts",
+  "description": "",
+  "scripts": {
+	"local": "npm run envs && npm run set-local-command && npm run local:compose-up",
+	"local:dev": "npm run envs && npm run set-local-dev-command && npm run local:compose-up",
+	"populate-db": "docker exec greenpress_greenpress_1 npm run populate-db",
+	"stop": "docker rm $(docker ps -f name=greenpress -aq) -f",
```

Comment:
on windows this command sends err :
unknown shorthand flag: 'a' in -aq)

Reply:
I committed a fix. Please check now.